lxc/config.go

```diff
@@ -60,7 +60,7 @@ func (c *configCmd) usage() string {
 			"lxc config unset key                                   Unset server configuration key\n" +
 			"lxc config show [remote:]<container>                   Show container configuration\n" +
 			"lxc config trust list [remote]                         List all trusted certs.\n" +
-			"lxc config trust add [remote] [certfile.crt]           Add certfile.crt to trusted hosts.\n" +
+			"lxc config trust add [remote] <certfile.crt>           Add certfile.crt to trusted hosts.\n" +
```

Comment:
Just updating to make it match the spec.

What would "lxc config trust add" without any argument mean? It can't possibly at a certificate to the trust store without being provided a certificate.